tests/test_awslambda/test_lambda.py

```diff
@@ -27,21 +29,23 @@ def get_test_zip_file1():
 def lambda_handler(event, context):
     return event
 """
-    return _process_lamda(pfunc)
+    return _process_lambda(pfunc)
```

Comment:
oh, nice catch!

Reply:
ya did a double take until I realized it :)